docs/security.md

```diff
@@ -0,0 +1,68 @@
+# Securing KUDO Cassandra Operator instances
+
+The KUDO Cassandra service supports Cassandra’s native transport **encryption**
+mechanism. The service provides automation and orchestration to simplify the use
+of these important features. For more information on Cassandra’s security, read

```

Comment:
Let's also rephrase "Cassadra" instances to "Apache Cassandra" for the same reason.
```suggestion
of these important features. For more information on Cassandra’s security, read

```